src/screens/SendToken/SendTokenConfirm.js

```diff
@@ -152,6 +155,22 @@ class SendTokenContacts extends React.Component<Props, State> {
               }
             />
             }
+            {(!!recipientUsername && Platform.OS === 'ios') &&
+            <MultilineTextInput
```

Comment:
Is it possible to join `TextInput` and `MultilineTextInput ` components into one and put a platform condition inside? Then it will be easier to re-use